.rubocop.yml

```diff
@@ -2,7 +2,7 @@ inherit_from: .rubocop_todo.yml
 
 require: rubocop-rspec
 AllCops:
-  TargetRubyVersion: 2.4
+  TargetRubyVersion: 2.5
```

Comment:
I'm sure you've already considered this but I figured I would ask:

This seems like a minor change but is there any impact on people using this gem with older Ruby versions?

It's only Rubocop also, so I would expect the operational impact to be very minimal but I figured I would ask anyhow, just in case.

Reply:
I think it's fine to change this as it's more of our internal. Also we have dropped support of 2.4 so it's fine to use 2.5 here.